test/controllers/VatDetailsControllerSpec.scala

```diff
@@ -263,20 +263,36 @@ class VatDetailsControllerSpec extends ControllerBaseSpec {
     }
   }
 
-  "Calling .detailsRedirectToEmailVerification" should {
+  "Calling .detailsRedirectToEmailVerification" when {
 
-    "redirect to email verification" when {
+    "all relevant information is returned from vat-subscription" when {
 
-      "all relevant information is returned from vat-subscription" in {
+      "there is no pending PPOB change" should {
```

Comment:
Can you add one extra test to each of these scenarios (no pending & pending) to check the `inFlightContactKey` is being added or not added to session, as that seems to be the difference between these two scenarios.

Bonus comment - can you remove the unused import in this file